### PR TITLE
pyjl: Migrate to plugins

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: pyjl/plugins.py [hunk 1]
```diff
@@ -129,32 +117,24 @@ def visit_asyncio_run(node, vargs) -> str:
...
+    # ::Int64 below is a hack to pass comb_sort.jl. Need a better solution
+    "floor": lambda n, vargs: f"Int64(floor({vargs[0]}))",
 }
 
 SMALL_USINGS_MAP = {
     "asyncio.run": "futures::executor::block_on",
 }
 
 DISPATCH_MAP = {
-    "max": functools.partial(JuiliaTranspilerPlugins.visit_min_max, is_max=True),
```
Comment (adsharma) [inline pyjl/plugins.py:153]: Add these back, so the tests you added pass for Julia?

**Review by adsharma — COMMENTED** [review]
Thanks for finishing up the plugin migration!

**Review by adsharma — COMMENTED** [review]
Looks good. Could help https://github.com/adsharma/py2many/issues/355

File: pyjl/plugins.py (hunk 1 above)
Comment (jayvdb) [reply pyjl/plugins.py:153]: The tests do pass; these were wrong and unnecessary copy and paste.

**Comment by jayvdb** [comment]
Please review all six recent PRs in my stack, as I need to rebase them all, again.

**Pull request merged**